Assets/Huawei/Demos/ML/LanguageDetection/LanguageDetectionDemoManager.cs

```diff
@@ -0,0 +1,188 @@
+using Assets.Huawei.Scripts.ML.LangDetector;
+using HuaweiMobileServices.ML.LanguageDetection;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class LanguageDetectionDemoManager : MonoBehaviour
+{
+
+    private const string Tag = "[HMS] LanguageDetectDemoManager ";
+    public Button onlineButton;
+    public Button offlineButton;
+    public InputField langInputField;
+    public GameObject detectionMenu;
+    public GameObject buttonMenu;
+    public Button firstBestDetectButton;
+    public Button probilityDetectButton;
+    public Button backButton;
+    public Text resultText;
+    public Text titleText;
+    // Start is called before the first frame update
+    void Start()
+    {
+        Debug.Log("AZIZ WAS HERE!!");
+        onlineButton.onClick.AddListener(OnOnlineButtonClicked);
+        offlineButton.onClick.AddListener(OnOfflineButtonClicked);
+        backButton.onClick.AddListener(OnBackButtonClicked);
+        langInputField.text = "Merhaba nasilsin, Hello how are you, مرحبا كيف حالك, 你好吗, Hallo hoe is het, Hola, cómo estás";
+    }
+
+
+    private void OnOnlineButtonClicked()
+    {
+        Debug.Log("Online clicked");
+        OnDetectionButtonClicked(DetectionMode.Online);
+        // Online butonuna basıldığında yapılacak işlemleri ekleyin
+    }
+
+    private void OnOfflineButtonClicked()
+    {
+        Debug.Log("Offline clicked");
```

Comment:
Can you add "TAG" to head of debug log that we needed ? Like;

`Debug.Log($"{TAG} Offline clicked");`

It is valid for all debug logs.